tasks/cache-total-supply.ts

```diff
@@ -45,6 +53,14 @@ task('cache-total-supply', 'Cache total supply on L2').setAction(
       );
       const ethValue = maxSubmissionCost.add(gasPriceBid.mul(maxGas));
 
+      if (taskArgs.estimatel2) {
+        console.log(`maxGas: ${maxGas.toString()}`);
+        console.log(`gasPriceBid: ${gasPriceBid.toString()}`);
+        console.log(`maxSubmissionCost: ${maxSubmissionCost.toString()}`);
+        console.log(`value: ${ethValue.toString()}`);
```

Comment:
would be more helpful to convert `ethValue` to gwei since this value is supposed to be used with a browser wallet which accepts gwei by default

Reply:
Left this as is for now because the `ethValue` in at least Etherscan is expected to be in terms of ETH and in order to account for different units required by different UIs printing in wei seems most convenient.